### DISCUSSION TITLE
How to use it?

### DISCUSSION THREAD

**Discussion by Preet-Sojitra** [toeverything/AFFiNE#236] [Q&A] [answered]
Since it is not ready for production use, so if I want to use it do I have to use it locally in my machine?

**Comment by duchenpaul** [comment]
See this #165, I think currently it is not ready to use for long term storage, the data is stored in the browser, which could be cleaned with brower housekeeping, looking forward the develops to rollout the local storage feature.

**Reply by Jengro777** [reply]
It is now ready to store data on the server side
https://github.com/toeverything/AFFiNE/discussions/461

docker pull ghcr.io/toeverything/affine:nightly-latest
docker run -itd -p 4200:3000 -v {your_path}:/app/data ghcr.io/toeverything/affine:nightly-server-latest

**Reply by fsj-digital** [reply]
hi there - pretty new to Affine but i love all i see: BTW - how to install it on Linux - i.e. on EndeavourOS!? 

Thx in advance

**Comment by ShortCipher5** [comment, marked as answer]
As the answer and steps for this are likely to continuously change and be updated, I would recommend anyone looking for an answer to visit the official docs for the latest information: [https://docs.affine.pro/](https://docs.affine.pro/)

**Reply by fsj-digital** [reply]
hi there - pretty new to Affine but i love all i see: BTW - how to install it on Linux - i.e. on EndeavourOS!? Thx in advance
